appspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.0
+os: linux
+files:
+  - source: /
+    destination: /var/www/html
+# permissions:
+#   - object: /var/www/html
+#     owner: deploy
+#     mode: 666
+#     type:
+#       - directory
```

Comment:
I have no clue what permissions do so not sure if this is something that needs to be left or if we can remove this to keep the file skinnier

Reply:
This is part my attempt to give AWS CodeDeploy permission to use the 'deploy' user and make sure that the 'deploy' user had proper permissions. It can be removed